src/components/Dialog/Dialog.stories.tsx

```diff
@@ -69,27 +69,31 @@ export default {
 
 const Template: StoryFn<DialogProps> = (args) => {
     const [isOpen, setIsOpen] = useToggleOverlay(false, { isBlockingModal: args.modality === Modality.BlockingModal });
+    const [showContent, setShowcontent] = useState(false);
```

Comment:
`C` should be capitalised in `setShowContent`

Reply:
I saw it and just fixed.